helmfile/overrides/tools/blazer.yaml.gotmpl

```diff
@@ -0,0 +1,136 @@
+applicationName: blazer
+
+ingress:
```

Comment:
To have the reverse proxy setup via nginx, we need to define the ingress, and no port forwarding via a service, if I got this correctly?

Reply:
You got it!